lib/nrf_inbuilt_key/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig NRF_INBUILT_KEY
+	bool "nRF9160 inbuilt key library"
```

Comment:
Can we hide this option for unrelated platforms somehow?

e.g. could we add a depends on nrf91 ?

Reply:
Do we need it also in this file? Wouldn't the selects of AT_CMD or BSD_LIBRARY indirectly depend on nrf91?

Reply:
I believe we do, what you could do to check is to open menuconfig from nrf52 and see if this option is presented as something that can be enabled.

Reply:
I guess to make this correct, we should probably depend on the real dependencies here, which is the BSD_LIBRARY and AT_CMD_PARSER. Not selecting anything but depend on what this library needs. Wouldn't that make sense?

Reply:
Sounds good to me.